### DISCUSSION TITLE
按照github配置后会出现 attempt to index a nil value

### DISCUSSION THREAD

**Discussion by zeroicey** [nshen/learn-neovim-lua#68] [Q&A]
![image](https://user-images.githubusercontent.com/59323300/180796665-6e82996b-0cb1-421a-99f5-4cbb5721541a.png)
![image](https://user-images.githubusercontent.com/59323300/180796744-9d078620-6c25-413a-a872-eb01e3a5e089.png)
![image](https://user-images.githubusercontent.com/59323300/180797695-70201eda-198b-49fe-85aa-42b3975ff739.png)
怎么解决啊

**Comment by nshen** [comment]
这个文件没找到吧
<img width="887" alt="image" src="https://user-images.githubusercontent.com/181506/180966568-b53c4b6f-840f-4b71-a835-affd5430a988.png">
